src/main/kotlin/com/github/oxc/project/oxcintellijplugin/actions/OxcFixAllAction.kt

```diff
@@ -24,13 +28,13 @@ class OxcFixAllAction : AnAction(), DumbAware {
 
     override fun actionPerformed(event: AnActionEvent) {
         val project = event.project ?: return
-        val editor = event.getData(CommonDataKeys.EDITOR) ?: return
+        val document = event.getOrCreateDocument() ?: return
 
         val notificationGroup = NotificationGroupManager.getInstance().getNotificationGroup("Oxc")
 
         val settings = OxcSettings.getInstance(project)
         val manager = FileDocumentManager.getInstance()
-        val virtualFile = manager.getFile(editor.document) ?: return
+        val virtualFile = manager.getFile(document) ?: return
```

Comment:
`manager.getFile(document)` is returning `null` for me when testing, so the action never runs.